app/views/layouts/admin.html.erb

```diff
@@ -26,7 +26,8 @@
 </head>
 <body class="admin default-header"
   data-honeybadger-key="<%= ApplicationConfig["HONEYBADGER_JS_API_KEY"] %>"
-  data-release-footprint="<%= ApplicationConfig["RELEASE_FOOTPRINT"] %>"
```

Comment:
**Confirmation Required** I'm not 100% sure if this key being referenced by an external app, so I need some confirmation here that it is fine to change the data key name. 

It made more sense to name it in accordance to what it references and consistent with the rest of the app. I've also added the latest commit id to the data keys on the body.